Vagrantfile

```diff
@@ -120,6 +120,12 @@ Vagrant.configure('2') do |config|
   Type "vagrant ssh" to login to the shell.
 MSG
 
+  # Temp fix for Vagrant 1.8.5
+  if Vagrant::VERSION =~ /^1.8.5/
```

Comment:
You could also require people to upgrade to at least version 1.8.5 by adding the following line:
`Vagrant.require_version '>= 1.8.5'`. Maybe interesting when 1.8.6 is out.